contracts/Core/VoteManager.sol

```diff
@@ -142,7 +142,7 @@ contract VoteManager is Initializable, ACL, VoteStorage, StateManager {
         return votes[stakerId].epoch;
     }
 
-    function getRandaoHash() public view returns (bytes32) {
+    function getRandaoHash() external view returns (bytes32) {
```

Comment:
I think `razor-go` needs this `getRandaoHash()` @ashish10677  Please clarify

Reply:
https://github.com/razor-network/razor-go/blob/e04f1727a4eaab155852dbf34cbbc3d54ddfdf33/utils/vote.go#L43